test/mbr_update_test.rb

```diff
@@ -20,6 +20,9 @@
         dev + num.to_s
       end
 
+      # by default common architecture"
+      allow(Yast::Arch).to receive(:architecture).and_return("x86_64")
```

Comment:
So this is staying because another method is querying `Arch.ppc`, and more methods do query `Arch.architecture` anyway. In the end I acknowledge that [`Arch.architecture`](http://www.rubydoc.info/github/yast/yast-yast2/Yast/ArchClass#architecture-instance_method) is public API after all, with boolean shortcuts. OK.